vars/virtualenv.groovy

```diff
@@ -0,0 +1,14 @@
+def create(String python, String path = "venv", boolean installRequirements = true) {
+  sh("virtualenv -p ${python} ${path}")
+  if (installRequirements && fileExists("requirements.txt")) {
+    run(path:path, command:"pip install -r requirements.txt")
+  }
+}
+
+
+def run(String path = "venv", String command) {
```

Comment:
I don't know how to interpret this... Default parameters should always be at the end, otherwise `run(foo)` and `run(foo, bar)` are not using `foo` the same way and :dizzy_face: